.github/workflows/main.yml

```diff
@@ -119,8 +119,11 @@ jobs:
       - name: Nessie Spark 3.4 / 2.12 Extensions test
         run: ./gradlew -DscalaVersion=2.13 :nessie:nessie-iceberg:nessie-spark-extensions-3.4_2.13:test :nessie:nessie-iceberg:nessie-spark-extensions-3.4_2.13:intTest --scan
```

Comment:
nit: unrelated to this PR, but there's a mismatch of scala version in the parameter and name (2 lines above).